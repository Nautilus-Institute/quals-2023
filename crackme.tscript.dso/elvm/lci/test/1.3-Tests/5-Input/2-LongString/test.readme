This test checks to see whether a program correctly accepts input via the GIMMEH
statement.  It provides a long string of text as input.
