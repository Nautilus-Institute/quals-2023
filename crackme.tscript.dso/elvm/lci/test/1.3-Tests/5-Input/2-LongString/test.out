Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean viverra turpis et ligula semper egestas. Suspendisse volutpat gravida sagittis. Fusce accumsan, diam in mollis sagittis, libero felis sollicitudin lorem, sagittis vestibulum est enim eget ipsum. Pellentesque metus lectus, ultrices eu rutrum a, ornare quis lacus. Maecenas eleifend lorem et dui tempor tincidunt. Phasellus pretium eleifend eros, et pulvinar ipsum aliquet quis. Vestibulum pulvinar ipsum non felis congue nec dictum nisl consequat. Donec suscipit mollis dapibus. Aliquam mattis interdum pellentesque. In hac habitasse platea dictumst. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum vestibulum feugiat sapien at pulvinar. Pellentesque egestas quam sed massa pretium a faucibus enim dictum. Vivamus sodales viverra risus ut commodo. Etiam id leo a ligula condimentum pretium. Nulla in lorem dolor, eget laoreet urna. Donec vestibulum mattis vestibulum. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Sed orci massa, laoreet id luctus id, congue id diam. Phasellus consequat massa non nunc euismod sagittis suscipit sapien vestibulum. Integer a nibh dolor, quis luctus quam. Nullam faucibus cursus risus at elementum. Fusce nec elit eget ante lacinia luctus. Curabitur consequat, ante sed hendrerit tempus, enim ante fermentum nisl, posuere laoreet mi eros suscipit quam. Vivamus quis arcu dolor. Etiam ut elit neque. Proin ut tortor leo. Pellentesque mollis, libero non ullamcorper eleifend, nulla odio dapibus libero, et lobortis odio lorem et arcu. Morbi vel malesuada ligula. Nunc suscipit pretium ipsum, non tempus ante euismod molestie. Integer facilisis, lectus cursus cursus adipiscing, felis mauris luctus odio, sed interdum mi mauris id sapien. Sed egestas eros leo, at interdum quam. Donec vitae ipsum massa. Phasellus auctor ante at tortor fringilla at gravida ligula mattis. Donec vel massa enim. Duis ac arcu eget urna accumsan dapibus vitae eu sapien. Duis ornare, justo eget porttitor tincidunt, mauris eros ornare massa, eu congue est sapien vitae urna. Nunc nec erat commodo sem varius consectetur eget eu purus. Maecenas ac suscipit erat. Vivamus varius nisl sit amet risus aliquam id eleifend magna ornare. Aliquam ac dolor nunc, condimentum congue elit. Donec porta arcu odio. Vivamus aliquet, enim a rutrum porttitor, urna libero porta ipsum, eu condimentum ligula dui sit amet augue. Suspendisse potenti. Fusce sollicitudin magna eget odio posuere sollicitudin. Quisque mauris lorem, luctus ac tempor nec, iaculis ut justo. In sed porta magna. Etiam sapien dui, bibendum id vestibulum vitae, dictum ac nunc. Integer posuere tincidunt turpis, sit amet volutpat tellus semper et. Nullam at libero eu sapien ultricies pretium sed vitae nulla. Nunc non erat nisl, in porta odio. Nam urna nulla, imperdiet sit amet faucibus in, sollicitudin laoreet mi. Suspendisse nisl velit, ullamcorper vel interdum eget, semper et est. Morbi et ultrices dolor. Donec ante justo, viverra in viverra dapibus, porta a nisl. Donec in eleifend magna. Mauris convallis ligula non dui blandit mattis. Fusce rhoncus, neque ut congue facilisis, elit ligula lacinia sem, sed varius dolor tortor ac nulla. Cras nisi nisl, varius in ultrices et, semper sed leo. Donec mauris felis, cursus accumsan cursus in, sollicitudin sit amet massa. Vivamus nibh sem, placerat in condimentum id, facilisis vel nunc. Nam dapibus velit ac mauris vestibulum at venenatis nisl aliquet. Nunc suscipit pellentesque dignissim. Suspendisse dignissim rhoncus dui eu ultrices. Integer at quam at eros gravida consequat. Donec gravida sollicitudin sagittis. Duis semper lobortis sem ut laoreet. Phasellus laoreet aliquam quam, vel aliquam leo scelerisque quis. Proin hendrerit, nulla a rutrum congue, arcu eros malesuada arcu, at sollicitudin justo ipsum consequat ipsum. Aenean sapien lorem, imperdiet at ultricies at, iaculis sed leo.
